bazel-mode.el

```diff
@@ -21,10 +21,23 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	;; single and multiple line comment start
```

Comment:
Nit: remove "multiple line comment" since there are no multiline commends in Bazel

Reply:
Done. I'm not convinced these comments are too helpful but I'll keep them in for now.